### ISSUE TITLE
qemu user not member of qemu group, bridge helper broken

### ISSUE THREAD

**Issue by juliekoubova** [jirutka/qemu-openrc#25]
Ahoj Jakube, thank you for this great script and all the other work you're doing. This is not really an issue or feature request for `qemu-openrc`, I just likely don't understand how QEMU is supposed to be used on Alpine. Or maybe I messed up something along the way and my configuration is not as it should be.

Anyhoo, looking at the [qemu pre-install script](https://git.alpinelinux.org/aports/tree/community/qemu/qemu.pre-install), it seems  the `qemu` user is created with `kvm` as its primary group, and is never added to the `qemu` group. 

Does `qemu-openrc` do some OpenRC magic to run qemu with `qemu` group membership? Because when I simply `doas -u qemu /usr/lib/qemu/qemu-bridge-helper`, I get Permission denied.

If there isn't any magic involved, is the `qemu` user supposed to be used like this? Is it okay if I add it to the group? Should I report this to the qemu APK maintainer?

**Comment by juliekoubova** [comment]
(adding the user to the group seems to have fixed the bridge issue)

**Comment by jirutka** [comment]
Ahoj Julie, thank you a lot for your kind words and for sponsoring me!

> Does qemu-openrc do some OpenRC magic to run qemu with qemu group membership? 

No, it does not and there’s a bug – it doesn’t pass the `group` parameter to start-stop-daemon.

To be honest, I haven’t used qemu-openrc for a long time.

**Comment by juliekoubova** [comment]
> To be honest, I haven’t used qemu-openrc for a long time.

oh no, I want to use it for my home lab :D is there anything else you'd recommend? I really, really don't wanna use that XML insanity

**Comment by jirutka** [comment]
That doesn’t mean you cannot use it, it should still work.
I don’t know about anything similar.

**Comment by jirutka** [comment]
FYI: This bug is fixed in 0.11.1 which I released yesterday and also pushed to Alpine Linux edge and v3.18.

**Comment by juliekoubova** [comment]
It works great! I'm moving from my beloved FreeBSD for consistency, I already have Alpine on my desktop and tablet; FreeBSD doesn't really work as a desktop with my hardware, and Alpine seems the most BSD-y out of all the Linux distros 😄 I got really discouraged back when I saw everyone using the libvirt XMLs, but then I found qemu-openrc, which is very similar to vm-bhyve I was using on FreeBSD. So thanks again for that, and for the quick fix!
